osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -43,14 +43,7 @@ protected override void ParseStreamInto(StreamReader stream, T output)
                     continue;
                 }
 
-                try
-                {
-                    ParseLine(output, section, line);
-                }
-                catch (Exception e)
-                {
-                    Logger.Error(e, $"Failed to process line \"{line}\" into {output}");
```

Comment:
you are hiding information here. this used to tell you which line the failure happened on; now it doesn't.

Reply:
The new commit will show the message as it was before:
```
[runtime:important] 2019-08-07 10:48:53: Failed to process line "bad,event,this,should,fail" into osu.Game.Beatmaps.Beatmap: This is an exception
[runtime:important] 2019-08-07 10:48:53: Failed to process line "this,is,also,bad" into osu.Game.Beatmaps.Beatmap: This is an exception
```